monkey/infection_monkey/post_breach/clear_command_history/linux_clear_command_history.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+
+from infection_monkey.utils.environment import is_windows_os
+
+TEMP_HIST_FILE = '$HOME/monkey-temp-hist-file'
+
+
+def get_linux_commands_to_clear_command_history():
```

Comment:
This function is REALLY complicated since it returns a lot of different things which are only loosely related. There are two ways to fix this in my opinion:
1. Completely separate to 3 functions (which indicates these things are somewhat unrelated)
2. Return a new data object (can be a simple dataclass) which represents a "row" of the data